core/src/include/Xios.hpp

```diff
@@ -28,34 +28,91 @@ class Xios : public Configured<Xios> {
     Xios();
     ~Xios();
 
-    void finalize();
+    void close_context_definition();
     void context_finalize();
+    void finalize();
     bool isInitialized();
 
     void configure() override;
     void configureServer();
-    void configureCalendar();
+    void configureCalendar(); // TODO?
```

Comment:
@TomMelt I noticed you added this method to the class definition but didn't implement it. Did you have something in mind?

Reply:
Ahh, I think I was going to split configure up further but I realised it wasn't necessary.

Feel free to delete this method.

Reply:
```suggestion
```

Reply:
Addressed in acc3b554.